From bf0a4ef8086183e5e13a98f00aae9288db3c398a Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Sun, 20 Nov 2022 20:20:44 +0800
Subject: [PATCH 21/56] Revert "drm/msm/dp: correct 1.62G link rate at
 dp_catalog_ctrl_config_msa()"

This reverts commit ef59819976da26e71b885cd43e2b5cf30da2d4d8.
---
 drivers/gpu/drm/msm/dp/dp_catalog.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/msm/dp/dp_catalog.c b/drivers/gpu/drm/msm/dp/dp_catalog.c
index 613348b022fe..2da6982efdbf 100644
--- a/drivers/gpu/drm/msm/dp/dp_catalog.c
+++ b/drivers/gpu/drm/msm/dp/dp_catalog.c
@@ -416,7 +416,7 @@ void dp_catalog_ctrl_config_msa(struct dp_catalog *dp_catalog,
 
 	if (rate == link_rate_hbr3)
 		pixel_div = 6;
-	else if (rate == 162000 || rate == 270000)
+	else if (rate == 1620000 || rate == 270000)
 		pixel_div = 2;
 	else if (rate == link_rate_hbr2)
 		pixel_div = 4;
-- 
2.25.1

